src/main/java/org/jilt/internal/AbstractBuilderGenerator.java

```diff
@@ -333,13 +334,19 @@ protected final TypeVariableName mangleTypeParameter(TypeVariableName typeVariab
     private ParameterSpec setterParameterSpec(VariableElement attribute, TypeName parameterType) {
         ParameterSpec.Builder ret = ParameterSpec.builder(parameterType,
                 this.attributeSimpleName(attribute));
-        AnnotationMirror nullableAnnotation = this.firstAnnotationCalledNullable(attribute);
-        if (nullableAnnotation != null) {
-            ret.addAnnotation(AnnotationSpec.get(nullableAnnotation));
-        }
+        addAnnotations(ret, attribute.getAnnotationMirrors());
+        addAnnotations(ret, attribute.asType().getAnnotationMirrors());
```

Comment:
Let's prefix the calls with `this`:

```suggestion
        this.addAnnotations(ret, attribute.getAnnotationMirrors());
        this.addAnnotations(ret, attribute.asType().getAnnotationMirrors());
```